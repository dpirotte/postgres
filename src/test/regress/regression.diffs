diff -U3 /home/admin/src/postgres/src/test/regress/expected/subscription.out /home/admin/src/postgres/src/test/regress/results/subscription.out
--- /home/admin/src/postgres/src/test/regress/expected/subscription.out	2020-10-10 02:41:54.649505484 +0000
+++ /home/admin/src/postgres/src/test/regress/results/subscription.out	2020-11-20 04:19:18.770055960 +0000
@@ -76,10 +76,10 @@
 ERROR:  invalid connection string syntax: missing "=" after "foobar" in connection info string
 
 \dRs+
-                                                            List of subscriptions
-      Name       |           Owner           | Enabled | Publication | Binary | Streaming | Synchronous commit |          Conninfo           
------------------+---------------------------+---------+-------------+--------+-----------+--------------------+-----------------------------
- regress_testsub | regress_subscription_user | f       | {testpub}   | f      | f         | off                | dbname=regress_doesnotexist
+                                                                 List of subscriptions
+      Name       |           Owner           | Enabled | Publication | Binary | Messages | Streaming | Synchronous commit |          Conninfo           
+-----------------+---------------------------+---------+-------------+--------+----------+-----------+--------------------+-----------------------------
+ regress_testsub | regress_subscription_user | f       | {testpub}   | f      | f        | f         | off                | dbname=regress_doesnotexist
 (1 row)
 
 ALTER SUBSCRIPTION regress_testsub SET PUBLICATION testpub2, testpub3 WITH (refresh = false);
@@ -91,10 +91,10 @@
 ALTER SUBSCRIPTION regress_testsub SET (create_slot = false);
 ERROR:  unrecognized subscription parameter: "create_slot"
 \dRs+
-                                                                List of subscriptions
-      Name       |           Owner           | Enabled |     Publication     | Binary | Streaming | Synchronous commit |           Conninfo           
------------------+---------------------------+---------+---------------------+--------+-----------+--------------------+------------------------------
- regress_testsub | regress_subscription_user | f       | {testpub2,testpub3} | f      | f         | off                | dbname=regress_doesnotexist2
+                                                                      List of subscriptions
+      Name       |           Owner           | Enabled |     Publication     | Binary | Messages | Streaming | Synchronous commit |           Conninfo           
+-----------------+---------------------------+---------+---------------------+--------+----------+-----------+--------------------+------------------------------
+ regress_testsub | regress_subscription_user | f       | {testpub2,testpub3} | f      | f        | f         | off                | dbname=regress_doesnotexist2
 (1 row)
 
 BEGIN;
@@ -126,10 +126,10 @@
 ERROR:  invalid value for parameter "synchronous_commit": "foobar"
 HINT:  Available values: local, remote_write, remote_apply, on, off.
 \dRs+
-                                                                  List of subscriptions
-        Name         |           Owner           | Enabled |     Publication     | Binary | Streaming | Synchronous commit |           Conninfo           
----------------------+---------------------------+---------+---------------------+--------+-----------+--------------------+------------------------------
- regress_testsub_foo | regress_subscription_user | f       | {testpub2,testpub3} | f      | f         | local              | dbname=regress_doesnotexist2
+                                                                        List of subscriptions
+        Name         |           Owner           | Enabled |     Publication     | Binary | Messages | Streaming | Synchronous commit |           Conninfo           
+---------------------+---------------------------+---------+---------------------+--------+----------+-----------+--------------------+------------------------------
+ regress_testsub_foo | regress_subscription_user | f       | {testpub2,testpub3} | f      | f        | f         | local              | dbname=regress_doesnotexist2
 (1 row)
 
 -- rename back to keep the rest simple
@@ -162,19 +162,19 @@
 CREATE SUBSCRIPTION regress_testsub CONNECTION 'dbname=regress_doesnotexist' PUBLICATION testpub WITH (connect = false, binary = true);
 WARNING:  tables were not subscribed, you will have to run ALTER SUBSCRIPTION ... REFRESH PUBLICATION to subscribe the tables
 \dRs+
-                                                            List of subscriptions
-      Name       |           Owner           | Enabled | Publication | Binary | Streaming | Synchronous commit |          Conninfo           
------------------+---------------------------+---------+-------------+--------+-----------+--------------------+-----------------------------
- regress_testsub | regress_subscription_user | f       | {testpub}   | t      | f         | off                | dbname=regress_doesnotexist
+                                                                 List of subscriptions
+      Name       |           Owner           | Enabled | Publication | Binary | Messages | Streaming | Synchronous commit |          Conninfo           
+-----------------+---------------------------+---------+-------------+--------+----------+-----------+--------------------+-----------------------------
+ regress_testsub | regress_subscription_user | f       | {testpub}   | t      | f        | f         | off                | dbname=regress_doesnotexist
 (1 row)
 
 ALTER SUBSCRIPTION regress_testsub SET (binary = false);
 ALTER SUBSCRIPTION regress_testsub SET (slot_name = NONE);
 \dRs+
-                                                            List of subscriptions
-      Name       |           Owner           | Enabled | Publication | Binary | Streaming | Synchronous commit |          Conninfo           
------------------+---------------------------+---------+-------------+--------+-----------+--------------------+-----------------------------
- regress_testsub | regress_subscription_user | f       | {testpub}   | f      | f         | off                | dbname=regress_doesnotexist
+                                                                 List of subscriptions
+      Name       |           Owner           | Enabled | Publication | Binary | Messages | Streaming | Synchronous commit |          Conninfo           
+-----------------+---------------------------+---------+-------------+--------+----------+-----------+--------------------+-----------------------------
+ regress_testsub | regress_subscription_user | f       | {testpub}   | f      | f        | f         | off                | dbname=regress_doesnotexist
 (1 row)
 
 DROP SUBSCRIPTION regress_testsub;
@@ -185,19 +185,19 @@
 CREATE SUBSCRIPTION regress_testsub CONNECTION 'dbname=regress_doesnotexist' PUBLICATION testpub WITH (connect = false, streaming = true);
 WARNING:  tables were not subscribed, you will have to run ALTER SUBSCRIPTION ... REFRESH PUBLICATION to subscribe the tables
 \dRs+
-                                                            List of subscriptions
-      Name       |           Owner           | Enabled | Publication | Binary | Streaming | Synchronous commit |          Conninfo           
------------------+---------------------------+---------+-------------+--------+-----------+--------------------+-----------------------------
- regress_testsub | regress_subscription_user | f       | {testpub}   | f      | t         | off                | dbname=regress_doesnotexist
+                                                                 List of subscriptions
+      Name       |           Owner           | Enabled | Publication | Binary | Messages | Streaming | Synchronous commit |          Conninfo           
+-----------------+---------------------------+---------+-------------+--------+----------+-----------+--------------------+-----------------------------
+ regress_testsub | regress_subscription_user | f       | {testpub}   | f      | f        | t         | off                | dbname=regress_doesnotexist
 (1 row)
 
 ALTER SUBSCRIPTION regress_testsub SET (streaming = false);
 ALTER SUBSCRIPTION regress_testsub SET (slot_name = NONE);
 \dRs+
-                                                            List of subscriptions
-      Name       |           Owner           | Enabled | Publication | Binary | Streaming | Synchronous commit |          Conninfo           
------------------+---------------------------+---------+-------------+--------+-----------+--------------------+-----------------------------
- regress_testsub | regress_subscription_user | f       | {testpub}   | f      | f         | off                | dbname=regress_doesnotexist
+                                                                 List of subscriptions
+      Name       |           Owner           | Enabled | Publication | Binary | Messages | Streaming | Synchronous commit |          Conninfo           
+-----------------+---------------------------+---------+-------------+--------+----------+-----------+--------------------+-----------------------------
+ regress_testsub | regress_subscription_user | f       | {testpub}   | f      | f        | f         | off                | dbname=regress_doesnotexist
 (1 row)
 
 DROP SUBSCRIPTION regress_testsub;
