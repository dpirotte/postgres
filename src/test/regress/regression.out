test tablespace                   ... ok          102 ms
parallel group (20 tests):  int2 text oid name char txid money boolean pg_lsn varchar regproc int4 uuid float8 float4 int8 bit enum numeric rangetypes
     boolean                      ... ok           25 ms
     char                         ... ok           21 ms
     name                         ... ok           18 ms
     varchar                      ... ok           28 ms
     text                         ... ok           13 ms
     int2                         ... ok           12 ms
     int4                         ... ok           29 ms
     int8                         ... ok           39 ms
     oid                          ... ok           14 ms
     float4                       ... ok           34 ms
     float8                       ... ok           31 ms
     bit                          ... ok           45 ms
     numeric                      ... ok          120 ms
     txid                         ... ok           21 ms
     uuid                         ... ok           31 ms
     enum                         ... ok           45 ms
     money                        ... ok           22 ms
     rangetypes                   ... ok          157 ms
     pg_lsn                       ... ok           24 ms
     regproc                      ... ok           27 ms
parallel group (20 tests):  line numerology time path lseg timetz macaddr8 macaddr point circle xid inet tstypes interval date strings box polygon timestamp timestamptz
     strings                      ... ok           68 ms
     numerology                   ... ok           17 ms
     point                        ... ok           29 ms
     lseg                         ... ok           21 ms
     line                         ... ok           12 ms
     box                          ... ok          100 ms
     path                         ... ok           18 ms
     polygon                      ... ok          115 ms
     circle                       ... ok           30 ms
     date                         ... ok           39 ms
     time                         ... ok           16 ms
     timetz                       ... ok           23 ms
     timestamp                    ... ok          327 ms
     timestamptz                  ... ok          339 ms
     interval                     ... ok           38 ms
     inet                         ... ok           34 ms
     macaddr                      ... ok           27 ms
     macaddr8                     ... ok           24 ms
     tstypes                      ... ok           33 ms
     xid                          ... ok           31 ms
parallel group (10 tests):  unicode expressions comments misc_sanity horology type_sanity geometry oidjoins opr_sanity regex
     geometry                     ... ok           48 ms
     horology                     ... ok           38 ms
     regex                        ... ok          248 ms
     oidjoins                     ... ok           67 ms
     type_sanity                  ... ok           47 ms
     opr_sanity                   ... ok          149 ms
     misc_sanity                  ... ok           19 ms
     comments                     ... ok           14 ms
     expressions                  ... ok           11 ms
     unicode                      ... ok            6 ms
test create_function_1            ... ok            6 ms
test create_type                  ... ok           10 ms
test create_table                 ... ok          146 ms
test create_function_2            ... ok            5 ms
parallel group (5 tests):  copyselect copydml insert_conflict insert copy
     copy                         ... ok          182 ms
     copyselect                   ... ok           11 ms
     copydml                      ... ok           15 ms
     insert                       ... ok          118 ms
     insert_conflict              ... ok           60 ms
parallel group (3 tests):  create_operator create_procedure create_misc
     create_misc                  ... ok           47 ms
     create_operator              ... ok           11 ms
     create_procedure             ... ok           16 ms
parallel group (5 tests):  index_including create_view index_including_gist create_index_spgist create_index
     create_index                 ... ok          356 ms
     create_index_spgist          ... ok          233 ms
     create_view                  ... ok          113 ms
     index_including              ... ok           82 ms
     index_including_gist         ... ok          175 ms
parallel group (16 tests):  create_cast errors hash_func create_function_3 create_aggregate drop_if_exists roleattributes select typed_table create_am constraints vacuum infinite_recurse updatable_views inherit triggers
     create_aggregate             ... ok           33 ms
     create_function_3            ... ok           28 ms
     create_cast                  ... ok            9 ms
     constraints                  ... ok           78 ms
     triggers                     ... ok          306 ms
     select                       ... ok           43 ms
     inherit                      ... ok          275 ms
     typed_table                  ... ok           54 ms
     vacuum                       ... ok          178 ms
     drop_if_exists               ... ok           35 ms
     updatable_views              ... ok          257 ms
     roleattributes               ... ok           41 ms
     create_am                    ... ok           54 ms
     hash_func                    ... ok           26 ms
     errors                       ... ok           16 ms
     infinite_recurse             ... ok          182 ms
test sanity_check                 ... ok           59 ms
parallel group (20 tests):  select_distinct_on delete random select_having case namespace select_implicit prepared_xacts select_into select_distinct transactions portals union subselect arrays update hash_index join aggregates btree_index
     select_into                  ... ok           54 ms
     select_distinct              ... ok           84 ms
     select_distinct_on           ... ok           14 ms
     select_implicit              ... ok           49 ms
     select_having                ... ok           23 ms
     subselect                    ... ok          130 ms
     union                        ... ok           93 ms
     case                         ... ok           35 ms
     join                         ... ok          247 ms
     aggregates                   ... ok          261 ms
     transactions                 ... ok           89 ms
     random                       ... ok           22 ms
     portals                      ... ok           90 ms
     arrays                       ... ok          152 ms
     btree_index                  ... ok          606 ms
     hash_index                   ... ok          210 ms
     update                       ... ok          169 ms
     delete                       ... ok           17 ms
     namespace                    ... ok           38 ms
     prepared_xacts               ... ok           49 ms
parallel group (20 tests):  init_privs drop_operator security_label tablesample lock object_address replica_identity collate password identity generated matview groupingsets brin rowsecurity spgist privileges gist gin join_hash
     brin                         ... ok          276 ms
     gin                          ... ok          514 ms
     gist                         ... ok          416 ms
     spgist                       ... ok          401 ms
     privileges                   ... ok          413 ms
     init_privs                   ... ok           24 ms
     security_label               ... ok           54 ms
     collate                      ... ok          154 ms
     matview                      ... ok          252 ms
     lock                         ... ok           80 ms
     replica_identity             ... ok          120 ms
     rowsecurity                  ... ok          344 ms
     object_address               ... ok          100 ms
     tablesample                  ... ok           68 ms
     groupingsets                 ... ok          264 ms
     drop_operator                ... ok           43 ms
     password                     ... ok          162 ms
     identity                     ... ok          183 ms
     generated                    ... ok          233 ms
     join_hash                    ... ok          731 ms
parallel group (13 tests):  async dbsize alter_operator collate.icu.utf8 tid tsrf tidscan misc_functions alter_generic sysviews misc incremental_sort create_table_like
     create_table_like            ... ok          112 ms
     alter_generic                ... ok           43 ms
     alter_operator               ... ok           14 ms
     misc                         ... ok           66 ms
     async                        ... ok            9 ms
     dbsize                       ... ok            9 ms
     misc_functions               ... ok           38 ms
     sysviews                     ... ok           57 ms
     tsrf                         ... ok           23 ms
     tid                          ... ok           20 ms
     tidscan                      ... ok           25 ms
     collate.icu.utf8             ... ok           19 ms
     incremental_sort             ... ok           69 ms
parallel group (6 tests):  amutils psql_crosstab collate.linux.utf8 psql rules stats_ext
     rules                        ... ok          143 ms
     psql                         ... ok           96 ms
     psql_crosstab                ... ok           14 ms
     amutils                      ... ok            8 ms
     stats_ext                    ... ok          475 ms
     collate.linux.utf8           ... ok           43 ms
test select_parallel              ... ok          342 ms
test write_parallel               ... ok           20 ms
parallel group (2 tests):  subscription publication
     publication                  ... ok           28 ms
     subscription                 ... FAILED       10 ms
parallel group (17 tests):  portals_p2 combocid advisory_lock xmlmap tsdicts guc equivclass functional_deps dependency select_views window tsearch cluster bitmapops indirect_toast foreign_data foreign_key
     select_views                 ... ok          105 ms
     portals_p2                   ... ok           18 ms
     foreign_key                  ... ok          360 ms
     cluster                      ... ok          171 ms
     dependency                   ... ok           57 ms
     guc                          ... ok           41 ms
     bitmapops                    ... ok          200 ms
     combocid                     ... ok           29 ms
     tsearch                      ... ok          171 ms
     tsdicts                      ... ok           39 ms
     foreign_data                 ... ok          248 ms
     window                       ... ok          134 ms
     xmlmap                       ... ok           32 ms
     functional_deps              ... ok           45 ms
     advisory_lock                ... ok           29 ms
     indirect_toast               ... ok          216 ms
     equivclass                   ... ok           42 ms
parallel group (6 tests):  jsonpath_encoding json_encoding jsonpath jsonb_jsonpath json jsonb
     json                         ... ok           38 ms
     jsonb                        ... ok           98 ms
     json_encoding                ... ok            7 ms
     jsonpath                     ... ok           12 ms
     jsonpath_encoding            ... ok            7 ms
     jsonb_jsonpath               ... ok           33 ms
parallel group (18 tests):  conversion prepare returning plancache xml limit copy2 temp polymorphism with sequence rowtypes truncate rangefuncs domain largeobject plpgsql alter_table
     plancache                    ... ok           69 ms
     limit                        ... ok           75 ms
     plpgsql                      ... ok          286 ms
     copy2                        ... ok           84 ms
     temp                         ... ok          103 ms
     domain                       ... ok          154 ms
     rangefuncs                   ... ok          146 ms
     prepare                      ... ok           44 ms
     conversion                   ... ok           23 ms
     truncate                     ... ok          141 ms
     alter_table                  ... ok          529 ms
     sequence                     ... ok          116 ms
     polymorphism                 ... ok          104 ms
     rowtypes                     ... ok          120 ms
     returning                    ... ok           46 ms
     largeobject                  ... ok          158 ms
     with                         ... ok          113 ms
     xml                          ... ok           68 ms
parallel group (9 tests):  hash_part partition_info reloptions explain indexing partition_prune partition_join partition_aggregate tuplesort
     partition_join               ... ok          361 ms
     partition_prune              ... ok          323 ms
     reloptions                   ... ok           29 ms
     hash_part                    ... ok           15 ms
     indexing                     ... ok          305 ms
     partition_aggregate          ... ok          363 ms
     partition_info               ... ok           28 ms
     tuplesort                    ... ok          546 ms
     explain                      ... ok           37 ms
test event_trigger                ... ok           40 ms
test fast_default                 ... ok           51 ms
test stats                        ... ok          548 ms
